assets/js/view/company.rb

```diff
@@ -2,6 +2,8 @@
 
 module View
   class Company < Snabberb::Component
+    include Actionable
```

Comment:
require this to be safe

Reply:
you mean add require 'view/actionable' on top?

Reply:
yes